src/Contracts/NebulaField.php

```diff
@@ -161,6 +161,11 @@ public function getRules(): array
         return $this->rules;
     }
 
+    public function shouldRender()
+    {
+        return !($this instanceof \LarsKlopstra\Nebula\Fields\HasMany);
```

Comment:
I'd import the namespace here instead of using FQN.

Reply:
Also, this bit of code will means that we need to use the details page to edit has many relationships. Is there no way that we can edit them on the edit page, or even the create page?

Reply:
Yeah, I copied this code from blade directly into the controller